lib/getWeekdayName.ts

```diff
@@ -1,15 +1,14 @@
 import { ChronosDate, ensureDate } from './helpers/ensureDate';
-import { LOCALE, LOCALE_OPTIONS, LocaleOptionsType } from './constants';
+import { LOCALE, LOCALE_WEEKDAY_OPTIONS } from './constants';
 
 /**
  * Get weekday name
  * @param value
  * @param format
  * @return - Weekday value
  */
-export default (value: ChronosDate, format = LOCALE_OPTIONS.LONG): string => {
+export default (value: ChronosDate, format: 'long' | 'short' = 'long'): string => {
   const date = ensureDate(value);
-  const weekdayFormat = format.toUpperCase();
```

Comment:
Now keys of object with types are written in lowercase, so we don't need this transformation.